### DISCUSSION TITLE
Encryption at rest

### DISCUSSION THREAD

**Discussion by dhatcher8** [git-lfs/git-lfs#5152] [Q&A] [answered]
Are files stored on git-lfs encrypted at rest?

I found the following about Github hosting with encrypted disks: "Source code stored on GitHub.com will be encrypted at rest, by default. Any source code previously stored on GitHub.com has been converted over to hosts with encrypted disks. For GitHub Enterprise Server customers, encryption at rest is dependent on the host in which Enterprise Server is running, not a function of the Server software itself."

I'm unsure if this similarly applies to lfs and if not I'm trying to determine if lfs data is encrypted at rest.

**Comment by bk2204** [comment, marked as answer]
Hey,

If you're asking about the Git LFS client, then that is dependent on the operating system on which the user is running.  If you're asking about GitHub, then you have to ask GitHub; this repository is only for the open source Git LFS client.
